solidity/test/bridge/Bridge.test.ts

```diff
@@ -892,30 +892,52 @@ describe("Bridge", () => {
 
                       it("should update the depositors balances", async () => {
                         // The sum of sweep tx inputs is 4148000 satoshi. The output
-                        // value is 4145001 so the fee is 2999. There is 5 deposits
-                        // so 599 satoshi fee should be incurred per deposit.
-                        // Each deposit should also incur the treasury fee whose
-                        // initial value is 0.05% of the deposited amount.
+                        // value is 4145001 so the sweep transaction fee is 2999.
+                        // There are 5 deposits so the fee per deposit is 599
+                        // and the indivisible remainder is 4 which means the
+                        // last deposit should incur 604 satoshi. Worth noting
```

Comment:
Shouldn't it be 603 instead of 604?

Reply:
Yep, right. Fixed https://github.com/keep-network/tbtc-v2/pull/164/commits/2e8a2b2f9a3a033fe4ddd8f76ba966394d839b8e